code/modules/projectiles/guns/energy/mindflayer.dm

```diff
@@ -6,4 +6,6 @@
 	projectile_type = /obj/item/projectile/beam/mindflayer
 	fire_sound = 'sound/weapons/Laser.ogg'
 	price_tag = 1500
-	gun_tags = list(GUN_LASER, GUN_ENERGY, GUN_SCOPE)
\ No newline at end of file
+	gun_tags = list(GUN_LASER, GUN_ENERGY, GUN_SCOPE)
+	twohanded = TRUE
```

Comment:
Hard disogree
this makes a stun based weapon useless as all it does is make you dizzy
/obj/item/projectile/beam/mindflayer
	name = "flayer ray"

/obj/item/projectile/beam/mindflayer/on_impact(atom/target)
	if(ishuman(target))
		var/mob/living/carbon/human/M = target
		M.confused += rand(5,8)

Well it does has normal laser damage/ap I think its rare enuff and unused a nerf to it is unjust